hermes/views.py

```diff
@@ -78,19 +87,57 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
+    ra = fields.String(required=True)
+    dec = fields.String(required=True)
+    date_observed = fields.String(required=True)
+    date_format = fields.String()
+    telescope = fields.String(required=True)
+    instrument = fields.String(required=True)
```

Comment:
What goes here specifically? Make/Model? My pet name for my camera? What if I'm not at a large professional observatory and my telescope/instrument are basically one and the same?

Reply:
Then it can be repeated I think. I think it can be whatever the user decides is useful information to provide to the community. For an amateur for example the could provide the make and model of the camera.

Reply:
should it be required?

Reply:
That is an astronomer user committee level question.

Reply:
I don't think it is onerous to require this and its good practice for users to provide it.